.github/workflows/slack-message-broker.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow can be used to notify Slack channels when workflow_run events occur.
+# We currently notify the plutus-core-dev and plutus-tools-dev channels whenever
+# the script-evaluation-test or benchmark workflows run.
+# TODO this must be turned into a standalone GitHub action so that it can be used 
+# for arbitrary channels and workflows. 
+name: Slack Message Broker
+on:
+  workflow_run:
+
+jobs:
+  slack-broker:
+    runs-on: [ubuntu-latest]
+
+    steps:
+
+      - name: Prepare Slack Message
+        uses: actions/github-script@v6
+        id: prepare-slack-message
+        if: contains(fromJSON('["script-evaluation-test", "benchmark"]'), github.event.workflow_job.name)
+        with:
+          script: | 
+            const name = "${{ github.event.workflow_job.name }}";
+            const url = "${{ github.event.workflow_job.html_url }}";
+            const status = "${{ github.event.workflow_job.status }}";
+            const emojy = { success: "✅", failure: "❌", cancelled: "✋" }[status];
+            const conclusion = "${{ github.event.workflow_job.conclusion }}";
+            const action = "${{ github.event.action }}";
+            const message = `${emojy} \`${name}\` *${action}* | *${status}* | *${conclusion}* 👉🏻 <${url}|view logs>`;
+            core.setOutput("message", message); 
+
+      - name: Notify Slack
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: my-private-channel
```

Comment:
@michaelpj @zliu41 
Again, I must push to master before I can test this for real.
It works locally btw.
If you approve, I'll run `script-evaluation-test` and `benchmark` and I'll open a new PR changing `my-private-channel` to `plutus-core-dev` (unless you want to create a new channel to recevi these kinds of notifications?)

Reply:
If we're going to post in `plutus-core-dev` then it needs to be for _exceptional_ events, i.e. failures. I'm unconvinced of the utility of notifying for successful runs: if you want to know about those, you can just go to the actions page on github.

Reply:
Or at least, that's my interpretation of the point of such messages. I guess we could have a channel that gets logs for all runs, but I would never look at it (seems strictly worse than going on github).

Reply:
THB this was originally meant just for the script-evaluation-test, which runs daily, and for that it would be nice to know when it has succeeded/failed, but also that is has started at all.
We can actually remove Benchmark from the list here

Reply:
So I guess my feeling is that for any workflow which doesn't report its status via a commit status or similar, it would be nice to have a failure notification in the channel. Because those are the things we're going to miss. For the benchmarking job I guess it's less of a problem now that it posts a comment with a link to the workflow run, but maybe it would still be useful to see failures.